erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -309,6 +309,9 @@ def set_pos_fields(self, for_validate=False):
 		if not self.pos_profile:
 			pos_profile = get_pos_profile(self.company) or {}
 			self.pos_profile = pos_profile.get('name')
+			self.write_off_amount_of_pos_profile = pos_profile.get('write_off__amount')
+			self.write_off_amount_can_not_be_more_than_positive_or_negative = pos_profile.get('write_off_amount_can_not_be_more_than_positive_or_negative')
+			self.outstanding_amount_has_to_be_0 = pos_profile.get('outstanding_amount_has_to_be_0')
```

Comment:
outstanding_amount_has_to_be_0 is not available in the pos_profile.json, if you have added this as new field there is no changes showing in the file pos_profile.json

Reply:
Fixed